package-lock.json

```diff
@@ -1,12834 +1,8 @@
 {
     "name": "cla-assistant",
     "version": "1.8.3",
-    "lockfileVersion": 2,
```

Comment:
can you please use >=npm7 (`npm install -g npm@7`) to create it, to use lockFileVersion2. Otherwise we always switch between version 1 and 2.